test/functional/sqlsrv/srv_007_login_timeout.phpt

```diff
@@ -9,17 +9,17 @@ Intentionally provide an invalid server name and set LoginTimeout. Verify the ti
 
 $serverName = "WRONG_SERVER_NAME";
 
-$t0 = round(microtime(true));
+$t0 = microtime(true);
 
-$conn = sqlsrv_connect( $serverName , array("LoginTimeout" => 8));
+$conn = sqlsrv_connect($serverName , array("LoginTimeout" => 8));
```

Comment:
try combination with ColumnEncryption keywords as well

Reply:
I'll pass this one... just trying to understand its intermittent failures